org-code-javabuilder/theater/src/main/java/org/code/theater/Stage.java

```diff
@@ -45,13 +91,21 @@ public void playNote(Instrument instrument, int note, double seconds) {}
    * @param seconds The number of seconds to wait. This can be a fraction of a second, but the
    *     smallest value can be .1 seconds.
    */
-  public void wait(double seconds) {}
+  public void pause(double seconds) {
+    this.gifWriter.writeToGif(this.image, (int) (seconds * 1000));
+  }
 
   /** Clear everything, starting from the beginning and emptying the canvas. */
-  public void reset() {}
+  public void reset() {
```

Comment:
Should we also reset `hasPlayed` here? Or do we only allow one `play` per program?

Reply:
We only allow one play per program, reset is really more of a clear image than a full reset. I will update the comment and check with curriculum that we want to keep this method name.

Reply:
full conversation is [here](https://codedotorg.slack.com/archives/C01EF4GJ9GE/p1624553158138800) short version is we are renaming to `clear` and adding a Color parameter for the new background color